ci/coverity/model_file.c

```diff
@@ -0,0 +1,88 @@
+#define LargestIntegralType unsigned long long
+
+void _assert_true(const LargestIntegralType result,
+                  const char* const expression,
```

Comment:
(style)  code indent should use tabs where possible
(style)  "foo* const bar" should be "foo * const bar"